src/applications/vaos/components/review/PreferredDates.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { formatDateLong } from '../../../../platform/utilities/date';
+
+function PreferredDates(props) {
+  const dates = props.dates?.map((selected, i) => (
+    <span key={i}>
```

Comment:
This should probably be marked up as `<li>` elements inside a `<ul>`.

Reply:
Done.